webservices/rest.py

```diff
@@ -154,6 +161,11 @@ def limit_remote_addr():
                 abort(403)
             if source_ip in BLOCKED_IPS:
                 abort(403)
+            if RESTRICT_DOWNLOADS not in falses and '/download/' in request.url:
+                # 'X-Api-User-Id' header is passed through by the API umbrella
+                request_api_key_id = request.headers.get('X-Api-User-Id')
+                if request_api_key_id != DOWNLOAD_WHITELIST_API_KEY_ID:
+                    abort(403)
 
 
```

Comment:
Looks great. One minor issue is that I am not 100% sure 'RESTRICT_DOWNLOADS' should be within 'FEC_API_WHITELIST_IPS' conditional block, or we should keep them in parallel logically? I guess we always set FEC_API_WHITELIST_IPS to true. 

Great thought and smart implementation on handling this download hacking issue:-)

Reply:
Thank you @qqss88! We can only protect downloads when the API umbrella is in place, which corresponds with `FEC_API_WHITELIST_IPS`. I think a better name might be `REQUIRE_API_UMBRELLA` or something like that. Thoughts?

Reply:
sure - that clear things up. And yes, that name make sense better:-)

Reply:
For now, this is holding up the release so I'll put in a new issue for the rename, since we need the `env var` to match.